Documentation/Signoffs/IdentificationSystem.md

```diff
@@ -0,0 +1,66 @@
+# Identification Subsystem  
+
+## Subsystem Function  
+
+This Subsystem's role in the team's design is to scan barcodes on school devices before being placed and when being taken from the machine. The subsystem also includes a RFID Card Reader to verify and store students' identity. The PC will then receive information from the modules and store information in a database for our customer. The Windows PC will also be able to control the visual components like LEDs through a PLC.
+
+## Constraints  
+
+* Barcode Scanners shall be low power and consume no greater than 5 V 0.5 A. This is due to being powered by the PC. Common computers only output a max rating of 5 V 0.5 A.  
+^ Source: https://resources.pcb.cadence.com/blog/2020-what-are-the-maximum-power-output-and-data-transfer-rates-for-the-usb-standards  
+
+* Barcode Scanners shall be able to decode Code-128 barcodes 
+
+* RFID Card Scanner shall be low power and consume no greater than 5 V 0.5 A   
+
+* RFID Card Scanner shall operate on 13.56 MHz commonly used for higher security access badges, like the school Eagle Cards  
+
+* A Barcode Scanner and Card Reader will be monitered by USB voltmeter to ensure correct operation  
+
+* Barcodes shall be of decent quality to follow ISO 15416 standard. Assures that the codes can be read by any scanner  
+
+* The PC shall run on Windows 10 to provide the needed applications for PLC and Database
+
+* The PC will provide an ethernet port to communicate with the PLC through a USB NIC (network interface card)  
+
+* The PC will provide at minimum an Intel Core i5-8400H or stronger CPU and 16GB RAM to support PLC programs
+
+## Schematic  
+
+![Schematic](https://github.com/DillonSW/Capstone_Team_5/blob/Team5-signoff-Barcode-Scanner/images/ID_System.jpg?raw=true)    
+
+The Schematic above shows the layout of the ID subsystem. The PC connects to the PLC through a USB NIC, which is a USB to ethernet adapter, and is powered by the power subsystem. The PC is connected to and allows for multiple modules through six USB 3.0 ports. Two USB Voltmeters are used for a Barcode Scanner and RFID Reader. The UI system is connected to the MiniPC through the HDMI on the PC.  
+
+## Analysis  
```

Comment:
Address these in the same order as the constraints and state what it is your are analyzing.